pom.xml

```
@@ -10,38 +10,36 @@
 
     <name>Restlet project</name>
     <description>Lightweight REST framework</description>
-    <url>http://www.restlet.org</url>
+    <url>https://restlet.talend.com/</url>
     <inceptionYear>2005</inceptionYear>
 
     <scm>
-        <url>http://restlet.tigris.org/svn/restlet/modules/</url>
+        <url>https://github.com/restlet/restlet-framework-java.git</url>
+        <connection>scm:git:${project.scm.url}</connection>
+        <developerConnection>scm:git:${project.scm.url}</developerConnection>
     </scm>
 
     <repositories>
         <repository>
             <id>maven-central</id>
             <name>Central Maven repository</name>
-            <url>https://repo1.maven.org/maven2/</url>
-        </repository>
-        <repository>
-            <id>maven2-java</id>
-            <name>Java.net repository</name>
-            <url>https://download.java.net/maven/2</url>
-        </repository>
-        <repository>
-            <id>maven1-java</id>
-            <url>https://download.java.net/maven/1</url>
-            <layout>legacy</layout>
+            <url>https://repo.maven.apache.org/maven2/</url>
         </repository>
         <repository>
             <id>maven-restlet</id>
             <name>Restlet repository</name>
-            <url>https://maven.restlet.org</url>
+            <url>https://maven.restlet.talend.com</url>
         </repository>
     </repositories>
 
     <properties>
+        <release>false</release>
+        <java.version>11</java.version>
```

do you mind to step back to version `8`? we changed our mind

```suggestion
        <java.version>1.8</java.version>
```